peloader/winexports.h

```diff
@@ -3,14 +3,14 @@
 
 extern struct hsearch_data crtexports;
 
-#define DECLARE_CRT_EXPORT(_name, _func)                    \
-    static void __constructor __const__ ## _func (void)     \
-    {                                                       \
-        ENTRY e = { _name, _func }, *ep;                    \
-        if (crtexports.table == NULL)                       \
-            hcreate_r(1024, &crtexports);                   \
-        hsearch_r(e, ENTER, &ep, &crtexports);              \
-        return;                                             \
+#define DECLARE_CRT_EXPORT(_name, _func)                                        \
+    static void __attribute__((constructor(101))) __const__ ## _func (void)     \
```

Comment:
Huh, I didn't know about this constructor priority syntax! Why do you it set to 101? (No complaint, just don't understand it!)

Reply:
Yep! I wanted the `DECLARE_CRT_EXPORT` constructors to be executed before the `ADD_CUSTOM_HOOK` ones. So I set priority 101 to the former and 102 to the latter. Why 101 and 102 because priorities from 0 to 100 are intended for reserved use, and gcc throws warning around if you pass priority <= 100.